reference/system-databases/inspection-result.md

```diff
@@ -0,0 +1,306 @@
+REFERENCE | consistent
+SEVERITY  | critical
+DETAILS   | the cluster has 2 different tidb version, execute the sql to see more detail: select * from information_schema.cluster_info where type='tidb'
+***************************[ 3. row ]***************************
+RULE      | threshold-check
+ITEM      | storage-write-duration
+TYPE      | tikv
+INSTANCE  | 172.16.5.40:23151
+VALUE     | 130.417
+REFERENCE | < 0.100
+SEVERITY  | warning
+DETAILS   | max duration of 172.16.5.40:23151 tikv storage-write-duration was too slow
+***************************[ 4. row ]***************************
+RULE      | threshold-check
+ITEM      | rocksdb-write-duration
+TYPE      | tikv
+INSTANCE  | 172.16.5.40:20151
+VALUE     | 108.105
+REFERENCE | < 0.100
+SEVERITY  | warning
+DETAILS   | max duration of 172.16.5.40:20151 tikv rocksdb-write-duration was too slow
+```
+
+You can have the following findings from the above diagnosis result:
+
+* The first line indicates that TiDB's `log.slow-threshold` configuration value is `0`, which might affect performance.
+* The second line indicates that two different TiDB versions exist in the cluster.
+* The third and fourth lines indicate that the TiKV write delay is too long, and the expected delay is no more than 0.1s. The actual delay is far beyond the expectation.
+
+Diagnose the cluster problem from "2020-03-26 00:03:00" to "2020-03-26 00:08:00". To specify the time range, you need to use the SQL Hint of `/*+ time_range() */`. See the following query example:
+
+{{< copyable "sql" >}}
+
+```sql
+select /*+ time_range("2020-03-26 00:03:00", "2020-03-26 00:08:00") */ * from inspection_result\G
+```
+
+```
+***************************[ 1. row ]***************************
+RULE      | critical-error
+ITEM      | server-down
+TYPE      | tidb
+INSTANCE  | 172.16.5.40:4009
+VALUE     |
+REFERENCE |
+SEVERITY  | critical
+DETAILS   | tidb 172.16.5.40:4009 restarted at time '2020/03/26 00:05:45.670'
+***************************[ 2. row ]***************************
+RULE      | threshold-check
+ITEM      | get-token-duration
+TYPE      | tidb
+INSTANCE  | 172.16.5.40:10089
+VALUE     | 0.234
+REFERENCE | < 0.001
+SEVERITY  | warning
+DETAILS   | max duration of 172.16.5.40:10089 tidb get-token-duration is too slow
+```
+
+You can have the following findings from the above result:
+
+* The first line indicates that the `172.16.5.40:4009` TiDB instance is restarted at `2020/03/26 00:05:45.670`.
+* The second line indicates that the maximum `get-token-duration` time of the `172.16.5.40:10089` TiDB instance is 0.234s, but the expected time is less than 0.001s.
+
+You can also specify conditions, for example, to query the `critical` level diagnosis results:
+
+{{< copyable "sql" >}}
+
+```sql
+select * from inspection_result where severity='critical';
+```
+
+Query only the diagnosis result of the `critical-error` rule:
+
+{{< copyable "sql" >}}
+
+```sql
+select * from inspection_result where rule='critical-error';
+```
```

Comment:
Please refer to suggestion for the previous similar section and update accordingly.